nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -228,9 +228,14 @@ let
 
   regInfo = hostPkgs.closureInfo { rootPaths = config.virtualisation.additionalPaths; };
 
+  # Optionally configure an XBOOTLDR partition if the corresponding mount point is set.
+  setupXbootldr = config.boot ? loader && config.boot.loader.systemd-boot.xbootldrMountPoint != null;
```

Comment:
In what circumstance is `config.boot ? loader` ever false?

Reply:
orfbog eval for nixos manual fails, for example.

Reply:
That's not normal, this can happen if you use the value of this inside of a manual-generated value I'd say.

Reply:
I'd rather get to the bottom of it than leave this incredibly strange, and likely wrong-ish condition.

Reply:
From what I can tell, the error is likely because the manual release code only loads a restricted set of modules that does not include the `boot.loader` module.

The solution I can think of is to introduce another config option in `qemu-vm.nix` instead of trying to directly use the config option from `systemd-boot.nix`.

Will need help with a better way if the above is not acceptable.

Reply:
@RaitoBezarius: I think the above is the only blocker. @JulienMalka already reviewed the other changes. If you or @ElvishJerricco can review the latest commit, this PR should be ready to merge.